static/app/views/replays/detail/header/replayDetailsUserBadge.tsx

```diff
@@ -25,6 +26,31 @@ export default function ReplayDetailsUserBadge({readerResult}: Props) {
   const replayRecord = readerResult.replayRecord;
   const replay = readerResult.replay;
 
+  const [isLive, setIsLive] = useState(replay?.getIsLive());
+
+  const computeIsLive = useCallback(() => replay?.getIsLive(), [replay]);
```

Comment:
Is this useCallback is necessary? We can have `replay` as a dep in the `useEffect` instead and call `replay.getIsLive()`.